config/locales/address/en_ET.yml

```diff
@@ -0,0 +1,24 @@
+en_ET:
+  helpers:
+    label:
+      facility:
+        name: "Name"
+        facility_type: "Facility Type *"
+        size: "Size *"
+        street_address: "House and road number"
```

Comment:
House and road number needs confirmation: A lot of districts don't have a house and road numbers
.